Makefile

```diff
@@ -234,6 +234,7 @@ $(ENVTEST): $(LOCALBIN)
 	test -s $(LOCALBIN)/setup-envtest || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
 
 .PHONY: bundle
+bundle: CRDDESC_OVERRIDE=:maxDescLen=0
```

Comment:
I forgot to add this on https://github.com/openstack-k8s-operators/cinder-operator/commit/e33a589d51de900c82bde5843cf18d7c43cf421a , thanks for adding it.